src/dyad/dtl/ucx_dtl.c

```diff
@@ -509,10 +555,8 @@ dyad_rc_t dyad_dtl_ucx_init (const dyad_ctx_t* ctx,
     //   * Auto initialization of request objects
     //   * Worker sleep, wakeup, poll, etc. features
     ucx_params.field_mask =
-        UCP_PARAM_FIELD_FEATURES | UCP_PARAM_FIELD_REQUEST_SIZE | UCP_PARAM_FIELD_REQUEST_INIT;
-    ucx_params.features = UCP_FEATURE_TAG |
-                          // UCP_FEATURE_RMA |
-                          UCP_FEATURE_WAKEUP;
+        UCP_PARAM_FIELD_FEATURES | UCP_PARAM_FIELD_REQUEST_SIZE;
+    ucx_params.features = UCP_FEATURE_RMA | UCP_FEATURE_AMO32;;
```

Comment:
Does this work for non RMA case?

Reply:
Added the TAG back. Somehow it got deleted. Oops